requirements.txt

```diff
@@ -33,7 +33,7 @@ psycopg2==2.5.1
 pytz==2014.10
 redis==2.9.0
 requests==2.7.0
-six==1.5.1
+six==1.6.0
```

Comment:
setuptools v34.1.1 (used for installation of django-appconf) required six>=1.6.0...changed to pinning setuptools version